taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/4_pitRmedhakANDam/10_sambhAvyAni_prAyashchittAni.md

```diff
@@ -3,17 +3,94 @@ title = "१० सम्भाव्यानि प्रायश्चित
+
+।। उत्तरीयशिलापात्रकर्तृद्रव्यविपर्यये-
+
+> पूर्वदत्ताञ्जलीन् पिंडान् पुनरित्याह गौतमः 
+
+इत्यादि वैद्यनाथीयवचनात् ॥ तत्क्रमश्चोक्तः कुंभकोणप्रयोगे । शिलानाशेऽन्यच्छिलात्रयं पूर्ववत्प्रतिष्ठाप्यावाह्य स्पृष्ट्वा जपेत् -
+
+> आयातु देवः सुमनाभिरूतिभिर्यमो हवेह प्रयताभिरक्ता । आसीदताग्ंसुप्रयतेहबर्हिष्यूर्जाय जात्यै मम शत्रुहत्यै
+[[TODO]] परिष्कार्यम्
+
+ इति ॥ 
+
+पूर्वदत्तान्वासोदकतिलोदकपिंडांश्च पुनर्दद्यात् ॥ ततो नष्टशिलानां पुनर्लाभे ता अप्यत्रैव संस्थाप्य उभयं स्पृष्ट्वा जपेत्-
+
+> यमे इव यतमाने यदैतं प्रंवा भरन्मानुषादेवयन्तः॥ आसीदतग्ंस्वमुलोकं विदाने स्वासस्थे भवतमिन्दवे नः 
+[[TODO]] परिष्कार्यम्
+
+इति ।। 
+
+शिलानां नाशे पुनर्दर्शने च लौकिकाग्नावाज्यं संस्कृत्य-
+
+
+> यमाय सोमग्ंसुनुत यमाय जुहुता हविः । यमग्ंह यज्ञो गच्छत्यग्निदूतो अरंकृतः स्वाहा ॥ यमाय घृतवद्धविर्जुहोत प्र च तिष्ठत । स नो देवेष्वायमद्दीर्घमायुः प्रजीवसे स्वाहा ॥ यमाय मधुमत्तमग्ंराज्ञे हव्यं जुहोतन ॥ इदं नम ऋषिभ्यः पूर्वजेभ्यः पूर्वेभ्यः पथिकृद्भ्यः स्वाहा
+[[TODO]] परिष्कार्यम्
+
+इति तिस्रः युवाहुतीर्जुहोति ॥ यमायेदं० ॥ 
+पिंडद्रव्यचरुस्थाल्योर्विपर्यासेऽन्यद्गृहीत्वा पूर्वदत्तपिंडान्वलींश्च पुनर्दद्यात् ॥ वस्त्रविपर्यासे पूर्वदत्तवासोदकादि सर्वं पुनर्दद्यात् ।। इति ॥
+
 ## दहनकाले श्मशानाऽग्निविनाशे
-॥ दहनकाले श्मशानाग्निनाशे--" दह्यमाने शवे वह्निर्नष्टो वर्षाभवेद्यदि । चितिकाष्ठं तु निर्मथ्य तेन प्रेतं पुनर्दहेत् ।। अभावे चितिकाष्टानां प्रायश्चित्ते कृतेऽग्निना । दाहयेत्तानि शिष्टानि शेषं वाप्सु विनिक्षिपेत् ” ॥
+
+॥ दहनकाले श्मशानाग्निनाशे--
+
+> दह्यमाने शवे वह्निर्नष्टो वर्षाभवेद्यदि । चितिकाष्ठं तु निर्मथ्य तेन प्रेतं पुनर्दहेत् ।। अभावे चितिकाष्ठानां प्रायश्चित्ते कृतेऽग्निना । दाहयेत्तानि शिष्टानि शेषं वाप्सु विनिक्षिपेत् ॥
+
 ## अस्थिनाशे
-॥ अस्थिनाशे – “पूर्णे दग्धे तथा कुर्याज्ज्वालौघेनास्थिनाशने । नदीतीरे सरितारे तन्मन्त्रांस्तु जपेइबुधः ।। अस्थिसञ्चयनात्पूर्व प्रमादापहृतं यदि । अस्थिप्रतिकृति- । कृत्य तद्देशान्मृदमाहरेत् ॥ अस्थिवच्चिनुयाद्विद्वान्लोष्ठं वा भस्म वा ततः । प्रमाणमस्थिवत्कृत्वा तन्मन्त्रांस्तु जपेद्बुधः" ॥ इत्यादि वचनैरस्थिनाशेऽग्न्यनुगतप्रायश्चित्तं कृत्वा तिस्रोऽवसर्जनीयाश्च हुत्वा भस्मादि सञ्चिनुयात् ॥ भस्माद्यभावे केवलं मन्त्रजपः ॥
+
+॥ अस्थिनाशे – 
+
+> पूर्णे दग्धे तथा कुर्याज्ज्वालौघेनास्थिनाशने । नदीतीरे सरित्तीरे तन्मन्त्रांस्तु जपेद्बुधः ।। अस्थिसञ्चयनात्पूर्वं प्रमादापहृतं यदि । अस्थिप्रतिकृतिं कृत्य तद्देशान्मृदमाहरेत् ॥ अस्थिवच्चिनुयाद्विद्वान्लोष्ठं वा भस्म वा ततः । प्रमाणमस्थिवत्कृत्वा तन्मन्त्रांस्तु जपेद्बुधः ॥ 
```

Comment:
श्लोकार्धा भिन्नपङ्क्तिषु स्युः।

Reply:
new lines within paragraph,): R202212 (https://youtu.be/opLrf8kAazA)
अत्रोक्त रीत्या कृतेऽपि भिन्नपङ्क्तौ एव दृश्यते

Reply:
प्रतिवचनं साधु लेखितम्।  
किञ्च, न स्वयम् एव "Resolve conversation" इति कीलकं नुदेत् - अहम् परीक्ष्य तत् कीलकं नुदामि।  
अपरञ्च, "श्लोकार्धा भिन्नपङ्क्तिषु स्युः।" इति दोषः, अन्यत्रास्यां सञ्चिकायाम् अधुनापि दृश्यते।   
सर्वं तत् समीकृत्यात्रोत्तरं लिखतु।

Reply:
इदानीं‌ कृतवानस्मि